packages/flutter/lib/src/widgets/shortcuts.dart

```diff
@@ -424,10 +424,18 @@ class ShortcutManager extends ChangeNotifier with Diagnosticable {
 ///       child: Focus(
 ///         autofocus:true,
 ///         child: Column(
+///           mainAxisAlignment: MainAxisAlignment.center,
 ///           children: <Widget>[
-///             Text('Add to the counter by pressing keyboard Shift and keyboard "K"'),
-///             Text('Subtract from the counter by pressing keyboard Shift and keyboard "L"'),
-///             Text('count: $count'),
+///             Text('Add: keyboard Shift + "k"'),
```

Comment:
For this one, the alignment is a little off and the text is a little too verbose. Also added some styling to differentiate where the user should look at for the actual state change.